build-tools/automation/azure-pipelines.yaml

```diff
@@ -258,7 +258,7 @@ stages:
         $vsixInstaller = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe"
         $ts = Get-Date -Format FileDateTimeUniversal
         $log = "xavsixdowngrade-$ts.log"
-        $process = Start-Process -NoNewWindow -FilePath $vsixInstaller -ArgumentList "/downgrade:Xamarin.Android.Sdk /admin /quiet /logFile:$log" -Wait -PassThru
+        $process = Start-Process -NoNewWindow -FilePath $vsixInstaller -ArgumentList "/downgrade:Xamarin.Android.Sdk /admin /quiet /logFile:$log" -Wait -PassThru -RedirectStandardError "err.txt"
```

Comment:
The `xavsixdowngrade` log should contain all relevant info, and it is being printed to stdout. We could upload this stderr file but I think it will only contain redundant information. This change was made to avoid the non-fatal and expected error message `Xamarin.Android SDK cannot be reverted` from shown in the build summary (as seen below):
https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=3881645&view=results

Reply:
I think you can put `-RedirectStandardError Out-Null` if you really want to throw it away.

Reply:
I tried to play around with this and ultimately decided to cat the file back as a precaution.